index.js

```diff
@@ -75,12 +75,15 @@ var SmartBanner = function (options) {
 	// - user is on mobile safari for ios 6 or greater (iOS >= 6 has native support for SmartAppBanner)
 	// - running on standalone mode
 	// - user dismissed banner
+	this.appMeta = mixins[this.type].appMeta;
```

Comment:
This statement breaks on desktop as this.type is undefined.

Reply:
@dy could you please look into this and make a new release?

Reply:
@Seldaek Hello. I've just tested inside test folder. What's the purpose of rendering smart-banner on desktop?

Reply:
There is no purpose of course but I don't wanna deal with device detection if the library is doing it anyway and does not show the banner when on desktop. It worked fine until now, but this generates an error.

Reply:
Hmm, I've looked through comments and found options when we should not render anything "device os is not supported". This options is based on device type, but it is placed after my statement. To fix current issue i would suggest to move type check up so when there is no appropriate type we won't construct SmartBanner.

Reply:
@Seldaek comment this one please https://github.com/kudago/smart-app-banner/pull/101